Sources/GutenbergKit/Sources/EditorViewController.swift

```diff
@@ -136,15 +144,30 @@ public final class EditorViewController: UIViewController, GutenbergEditorContro
     }
 
     private func getEditorConfiguration() -> WKUserScript {
-        let jsCode = """
+        let escapedTitle = initialTitle.addingPercentEncoding(withAllowedCharacters: .alphanumerics)!
+        let escapedContent = _initialRawContent.addingPercentEncoding(withAllowedCharacters: .alphanumerics)!
+        let postId = id != nil ? id! : nil
+        let hasThemeStylesEnabled = themeStyles != nil ? themeStyles! : false
+
+        var jsCode = """
         window.GBKit = {
             siteApiRoot: '\(siteApiRoot)',
             siteApiNamespace: '\(siteApiNamespace)',
-            authHeader: '\(authHeader)'
+            authHeader: '\(authHeader)',
+            themeStyles: \(hasThemeStylesEnabled)
         };
         localStorage.setItem('GBKit', JSON.stringify(window.GBKit));
         "done";
         """
+        if let postId = id {
```

Comment:
Does `postId` serve a specific purpose or could we rely upon `id` directly?

Reply:
I've changed the approach in https://github.com/wordpress-mobile/GutenbergKit/pull/21/commits/54aedfb4541fb8465a6b4caa2c2c58f1d86334a3 thanks!

Reply:
After reviewing your changes here, I now understand your original let guard clause. Feel free to revert the changes for these specific lines if you prefer the previous implementation. Either approach is fine with me.

Reply:
I think with the latest changes its ok because we are checking `id != nil` before setting the post data, and then unwrapping the value with `id!`